src/plugins/unified_search/public/query_string_input/text_based_languages_editor/resizable_button.scss

```
@@ -0,0 +1,79 @@
+
+.unifiedTextLangEditor--resizableButton {
+  position: relative;
+  flex-shrink: 0;
+  z-index: 1;
+  cursor: row-resize;
+  height: $euiSize;
+  margin-top: -($euiSize / 2);
+  margin-bottom: -($euiSize / 2);
+
+  &:before,
+  &:after {
+    content: '';
+    display: block;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    width: $euiSizeM;
+    height: 1px;
+    background-color: $euiColorDarkestShade;
+    transition: (
+      width $euiAnimSpeedFast ease,
+      height $euiAnimSpeedFast ease,
+      transform $euiAnimSpeedFast ease,
+      background-color $euiAnimSpeedFast ease
+    );
+  }
+
+  &:before {
+    transform: translate(-50%, -2px);
+  }
+
+  &:after {
+    transform: translate(-50%, 1px);
```

In order to achieve what I suggest above (i.e. moving the resize button), we'll likely want an additional 1px of space in between the faux drag handle so that the border can sit directly in the middle of the handle.

```suggestion
    transform: translate(-50%, 2px);
```